pset-iterators.js

```diff
@@ -108,6 +133,31 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = function(){
+
```

Comment:
@carly-ga good structured approach, an alternative approach would have been to handle the comparison logic without having to changing properties values to the original `presidents` array